torch_xla/test/test_utils.py

```diff
@@ -69,7 +69,9 @@ def get_summary_writer(logdir):
   if logdir:
     from tensorboardX import SummaryWriter
     writer = SummaryWriter(log_dir=logdir)
-    write_to_summary(writer, 0, dict_to_write={'StartupTime': 0})
+    current_time = time.time()
```

Comment:
No need to create a new local variable.

Reply:
ok - I get really paranoid when time.time() gets near an optional argument but I do think this case is OK